iphone/Classes/TiUIWebView.m

```diff
@@ -290,7 +290,10 @@ -(void)loadFile:(NSString*)absolutePath
 		textEncodingName:(NSString*)textEncodingName
 	   mimeType:(NSString*)mimeType
 {
-	NSURL *requestURL = [NSURL fileURLWithPath:absolutePath];
+    static unsigned long uniqueIdentifier = 0;
+    uniqueIdentifier++;
```

Comment:
@hansemannn I tried so many things to avoid this standard NSURLProtocol behaviour, but with no luck.
I've tried to select a "no-cache" policy in the request - no results.
I've tried to override the LocalProtocolHandler methods, but only one instance is created - so you can't do anything here.
I've checked that there aren't multithreading related issues and trying to avoid them using NSLock - no results.

After ~5 hours of debugging, this is the best solution I've found.
Let me know